components/sign-in-form.jsx

```diff
@@ -31,6 +31,15 @@ export default class SignInForm extends PureComponent {
         let user = formData.get('uname');
         let passwd = formData.get('passwd');
         let submit = document.getElementById('sign-in-submit');
+
+        // Reject invalid username format
+        if (!user.startsWith('@')) {
```

Comment:
Good quick fix but consider, if the username matches something like `.*:.*\..*`, then all that's missing is the first @ - might be good to just try with @ prepended

Reply:
Yes, simplifying the username format would be a good idea.